docs/overview.md

```diff
@@ -76,24 +76,24 @@ distributions.
 
 The table below shows the version of the components deployed by the playbook for ACS 7.x onward.
 
-| Component           | 23.2 Enterprise & Community | 7.4 Enterprise | 7.3 Enterprise | 7.2 Enterprise | 7.1 Enterprise |
+| Component           | 23.2 Enterprise (Community) | 7.4 Enterprise | 7.3 Enterprise | 7.2 Enterprise | 7.1 Enterprise |
 |---------------------|-----------------------------|----------------|----------------|----------------|----------------|
-| OpenJDK             | 17.0.9                      | 17.0.3         | 17.0.3         | 11.0.15        | 11.0.15        |
-| Apache Tomcat       | 10.1.20                     | 9.0.86         | 9.0.86         | 9.0.86         | 9.0.86         |
+| OpenJDK             | 17.0.11                     | 17.0.11        | 11.0.23        | 11.0.23        | 11.0.23        |
+| Apache Tomcat       | 10.1.25                     | 9.0.91         | 9.0.91         | 9.0.91         | 9.0.91         |
 | PostgreSQL          | 15.x                        | 14.x           | 14.x           | 13.x           | 13.x           |
 | Apache ActiveMQ     | 5.18.4                      | 5.17.6         | 5.17.6         | 5.16.7         | 5.16.7         |
-| Repository          | 23.2.1                      | 7.4.2          | 7.3.2          | 7.2.2          | 7.1.1.10       |
-| Share               | 23.2.1                      | 7.4.2          | 7.3.2          | 7.2.2          | 7.1.1.10       |
-| Search Services     | 2.0.10                      | 2.0.10         | 2.0.10         | 2.0.10         | 2.0.2.2        |
-| Search Enterprise   | 4.0.1 (Enterprise only)     | 3.3.3          | 3.2.1          | 3.1.1.1        | 3.1.1.1        |
```

Comment:
I liked the explicit `(Enterprise only)`, `(n/a)` doesn't help at all if you didn't notice also the column heading

Reply:
I understand but keeping the previous heading would end up with an odd column containing 2 versions separated with just a space